data-serving/scripts/export-data/transform.py

```diff
@@ -281,7 +253,7 @@ def convert_age(row: dict[str, Any], buckets: list[dict[str, Any]]) -> Optional[
 def convert_row(row: dict[str, Any], buckets: list[dict[str, Any]]) -> Optional[dict[str, Any]]:
     "Converts row for export using age buckets"
 
-    if "ObjectId" not in row["_id"]:
+    if not row["_id"].isdigit():
```

Comment:
Not a blocker for this, but we should re-visit ID as integers - can make it difficult to integrate external cases into the databases that also need to be updated, e.g. for mpox, we used `N` and `E` prefixes to integrate two different line lists into one.

Reply:
That makes sense, I'll create an issue for this.

Reply:
I think we should discuss this further. I would tend toward using a separate curation-specific field, as this seems to be a curation-specific action we would want to keep an audit trail for.